descriptors/ModuleDescriptor-template.json

```diff
@@ -1097,6 +1097,29 @@
       "description": "Ability to manage acquisition unit and update invoice"
     }
   ],
+  "metadata": {
+    "user": {
+      "type": "system",
+      "permissions": [
```

Comment:
please add these permissions as well:
"orders-storage.order-invoice-relationships.collection.get"
"orders-storage.order-invoice-relationships.item.post"
"acquisitions-units-storage.memberships.collection.get"

Also, looks like endpoints usage was changed and these permissions have started to be used: 
"acquisitions-units-storage.units.collection.get", "acquisitions-units-storage.memberships.collection.get".
And these are not required:
"acquisitions-units.memberships.collection.get", "acquisitions-units.units.collection.get".

Reply:
@RuslanLavrov  Done! can you double-check please?

Reply:
thanks!